src/pages/LandingPage/LandingPage.js

```diff
@@ -66,6 +66,16 @@ class LandingPage extends Component {
                 </div>
               </Link>
           </div>
+          <div className="w-full sm:w-1/2 md:w-1/3 lg:w-1/4 xl:w-1/6 mb-4">
```

Comment:
```suggestion
          <div className="w-full sm:w-1/2 md:w-1/3 lg:w-1/4 xl:w-1/6 mb-4 sm:ml-4">
```
This will just give a bit of space between this card and the card on the left when viewed on a device above the `small` breakpoint